### PR TITLE
show help when type subo

### PR DESCRIPTION
**Pull request by samderlust**

### CODE REVIEW

**Comment by jagger27** [comment]
Very weird behaviour from `spf13/cobra` here. Why does the root function run the function defined in the `create` `cobra.Command` struct? This patch does seem to work, though.

File: subo/root.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ Explore the available commands by running 'subo --help'`,
 		Use:   "create",
 		Short: "create a runnable or project",
 		Long:  `create a new Atmo project or WebAssembly runnable`,
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (cohix) [inline subo/root.go:29]: Should this RunE be added to other top-level commands like `compute`, `create`, and `deploy` below?

File: subo/main.go [hunk 2]
```diff
@@ -2,17 +2,11 @@ package main
...
 import (
 	"os"
-
-	"github.com/spf13/cobra"
 )
 
 func main() {
 	rootCmd := rootCommand()
 
-	rootCmd.RunE = func(cmd *cobra.Command, args []string) error {
```
Comment (cohix) [inline subo/main.go:12]: You can set this field on the `cmd` object that's created inside the `rootCommand` function instead, right below the `Long` description.

File: subo/root.go [hunk 3]
```diff
@@ -17,6 +17,12 @@ Subo is the full toolchain for using and managing Suborbital Development Platfor
 including building WebAssembly Runnables and Atmo projects.
 
 Explore the available commands by running 'subo --help'`,
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (jagger27) [inline subo/root.go:20]: This makes much more sense.

File: subo/root.go (hunk 1 above)
Comment (cohix) [reply subo/root.go:35]: @samderlust are you adding it to the others too?

Comment (samderlust) [reply subo/root.go:35]: yeah I can do that

**Review by cohix — APPROVED** [review]

**Review by jagger27 — CHANGES_REQUESTED** [review]
I don't think we actually need to add these.
```go
if len(args) == 0 {
    cmd.Help()
}
return nil
```
Cobra provides per command help for free already, it seems. I built `main` from source and made sure I had the right binary and it already does the right thing:

```
$ ./subo compute                                            *[main]
manage Suborbital Compute Network resources

Usage:
  subo compute [command]

Available Commands:
  create      create compute network resources
  deploy      deploy compute network resources

Flags:
  -h, --help   help for compute

Use "subo compute [command] --help" for more information about a command.
```

**Comment by jagger27** [comment]
A simpler fix is implemented in #89

**Pull request closed without merging**